app/presenters/avatar_presenter.rb

```diff
@@ -2,25 +2,25 @@
 # frozen_string_literal: true
 
 class AvatarPresenter < BasePresenter
-  DEFAULT_IMAGE = ActionController::Base.helpers.image_path("user/default.png")
+  def base_hash(with_default = false)
+    avatar = {
+      small:  small(with_default),
+      medium: medium(with_default),
+      large:  large(with_default)
+    }.compact
 
-  def base_hash
-    {
-      small:  small,
-      medium: medium,
-      large:  large
-    }
+    avatar unless avatar.empty?
   end
 
-  def small
-    image_url(:thumb_small) || DEFAULT_IMAGE
+  def small(with_default = false)
```

Comment:
Layout/SpaceAroundEqualsInParameterDefault: Surrounding space detected in default value assignment.